lib/components/dashboard/course-list/CourseList.tsx

```diff
@@ -174,24 +175,21 @@ const CourseList: FC<Props> = ({ mode }) => {
     const temp: Year = yearArr[sourceIndex];
     yearArr.splice(sourceIndex, 1);
     yearArr.splice(destIndex, 0, temp);
-    dispatch(updateSelectedPlan({ ...currentPlan, years: yearArr }));
     const yearIdArr: string[] = yearArr.map((year) => year._id);
     const body = {
       plan_id: currentPlan._id,
       year_ids: yearIdArr,
     };
-    fetch(getAPI(window) + '/years/changeOrder', {
-      method: 'PATCH',
-      headers: {
-        'Content-Type': 'application/json',
-        Authorization: `Bearer ${token}`,
-      },
-      body: JSON.stringify(body),
-    })
-      .then((resp) => {
-        if (!resp.ok) {
-          console.log('ERROR:', resp);
-        }
+    axios
+      .patch(getAPI(window) + '/years/changeOrder', body, {
+        headers: {
+          'Content-Type': 'application/json',
```

Comment:
<img width="435" alt="Screenshot 2023-04-07 at 11 38 16 PM" src="https://user-images.githubusercontent.com/90944895/230701541-be2f8160-2bc5-4961-9e9a-365e10bded2f.png">
???????

Reply:
has this been addressed?